time_sequence_prediction/generate_sine_wave.py

```diff
@@ -0,0 +1,12 @@
+import math
+import numpy as np
+import cPickle as pickle
```

Comment:
can you just use `torch.save` to dump these.

Also, import cPickle only works in python2 fyi